src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts

```diff
@@ -14,9 +14,9 @@ import { fireEvent } from "../../../../common/dom/fire_event";
 import { Config, ConfigEntity } from "../../cards/hui-glance-card";
 import { configElementStyle } from "./config-elements-style";
 
+import "./entity-selectors/hui-glance-entity-select";
```

Comment:
Let's make a subfolder for every card 
```suggestion
import "./glance-card/hui-glance-entity-select";
```